src/features/snap/components/SnapReport.tsx

```diff
@@ -0,0 +1,38 @@
+import { useState, type FunctionComponent } from 'react';
```

Comment:
the file naming should be ReportSnap

Reply:
I saw @satyajeetkolhapure file called `AccountReport`, that is why i create a similar name called `SnapReport` here.

Reply:
Just change file name to `ReportSnap`